js/views/modals/About/About.js

```diff
@@ -0,0 +1,100 @@
+import $ from 'jquery';
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import Story from './Story';
+import Contributors from './Contributors';
+import Donations from './Donations';
+import License from './License';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      removeOnRoute: false,
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+
+    this.tabViewCache = {};
+    this.tabViews = {
+      Story,
+      Contributors,
+      Donations,
+      License,
+    };
+
+    this.listenTo(app.router, 'will-route', () => {
```

Comment:
This block could also be removed. Settings has some specific reason to manage this.. but for the About modal, we could just let the BaseModal take care of this (and as I mentioned, that's as simple as removing the `removeOnRoute: true` from our options hash above).